io/src/test/java/org/commonjava/maven/ext/io/PomIOTest.java

```diff
@@ -140,4 +201,31 @@ public void testWriteModel()
         assertTrue( targetFile.exists() );
         assertEquals( sb, FileUtils.readFileToString( targetFile, model.getModelEncoding() ) );
     }
+
+    @Test
+    public void testAddModifiedBy()
+                    throws Exception
+    {
+        URL resource = PomIOTest.class.getResource( filename );
+        assertNotNull( resource );
+        File pom = new File( resource.getFile() );
+        assertTrue( pom.exists() );
+
+        File targetFile = folder.newFile( "target.xml" );
+        FileUtils.copyFile( pom, targetFile );
+
+        List<Project> projects = pomIO.parseProject( targetFile );
+        projects.get( 0 ).setExecutionRoot();
+
+        HashSet<Project> changed = new HashSet<>();
+        changed.add( projects.get( 0 ) );
+        pomIO.rewritePOMs( changed );
+
+        String s = FileUtils.readFileToString( targetFile, Charset.defaultCharset() );
+        assertEquals( StringUtils.countMatches(s, "Modified by POM Manipulation Extension" ), 1);
+
+        pomIO.rewritePOMs( changed );
+        s = FileUtils.readFileToString( targetFile, Charset.defaultCharset() );
+        assertEquals( StringUtils.countMatches(s, "Modified by POM Manipulation Extension" ), 1);
```

Comment:
Arguments here are reversed. Should be: expected, actual.

Reply:
Ok

Reply:
Ok